diff --git a/BUILD b/BUILD
index c825050b..6e6bc604 100644
--- a/BUILD
+++ b/BUILD
@@ -171,6 +171,7 @@ cc_library(
         "hwy/detect_compiler_arch.h",  # private
         "hwy/print.h",
     ],
+    includes = ["."],
     compatible_with = [],
     copts = COPTS,
     defines = DEFINES,
diff --git a/MODULE.bazel b/MODULE.bazel
index bd798803..5c767b53 100644
--- a/MODULE.bazel
+++ b/MODULE.bazel
@@ -1,6 +1,11 @@
-module(name = "highway", version = "1.2.0")
+module(
+    name = "highway",
+    version = "1.2.0",
+    compatibility_level = 1,
+)
 
-bazel_dep(name = "bazel_skylib", version = "1.3.0")
-bazel_dep(name = "googletest", version = "1.12.1")
-bazel_dep(name = "rules_cc", version = "0.0.4")
-bazel_dep(name = "rules_license", version = "0.0.4")
+bazel_dep(name = "bazel_skylib", version = "1.6.1")
+bazel_dep(name = "googletest", version = "1.15.2", repo_name = "com_google_googletest")
+bazel_dep(name = "rules_cc", version = "0.0.9")
+bazel_dep(name = "rules_license", version = "0.0.7")
+bazel_dep(name = "platforms", version = "0.0.10")
